front/app/api/reports/useReport.ts

```diff
@@ -0,0 +1,17 @@
+import { useQuery } from '@tanstack/react-query';
+import { CLErrors } from 'typings';
+import fetcher from 'utils/cl-react-query/fetcher';
+import reportsKeys from './keys';
+import { ReportResponse, ReportsKeys } from './types';
+
+const fetchReport = ({ id }: { id: string }) =>
```

Comment:
Minor. We could simplify to 
```suggestion
const fetchReport = (id: string) =>
```